android-components/components/feature/search/src/test/java/mozilla/components/feature/search/telemetry/ads/AdsTelemetryTest.kt

```diff
@@ -139,7 +139,7 @@ class AdsTelemetryTest {
         assertEquals(Component.FEATURE_SEARCH, facts[0].component)
         assertEquals(Action.INTERACTION, facts[0].action)
         assertEquals(AdsTelemetry.SERP_SHOWN_WITH_ADDS, facts[0].item)
-        assertEquals(telemetry.providerList[0].name, facts[0].value)
+        assertEquals("google.in-content.organic.none", facts[0].value)
```

Comment:
Why are we seeing a `none` here?

Reply:
This is because in the url provided for the test we didn't add a query parameter for the provider's partner code.

Reply:
Ok, the conditions in which this is none should be documented in the metrics docs.